google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -38,48 +36,48 @@ class AdminDirectoryClient(_base_client.BaseClient):
 
     API_NAME = 'admin'
 
-    def __init__(self, credentials=None, rate_limiter=None):
+    def __init__(self, configs, credentials=None, rate_limiter=None):
```

Comment:
The code here has shifted a bit so that no params are needed for constructing this.  Is this still intentional to allow params to be passed in?